test/test-api-service-route.js

```diff
@@ -499,4 +499,61 @@ describe('service route', function () {
 
   });
 
+  describe('loading all outages', function () {
+
+    describe('with an anonymous user', function () {
+      before(function (done) {
+        agent.get('/logout').expect(302, done);
+      });
+
+      it('should require auth', function (done) {
+        storage.flush_database(function () {
+          storage.addService(validService, function (err, id) {
+            assert.ifError(err);
+            superAgentAssertions.shouldReturnStatusCode(agent, {
+              url : API_ROOT + '/services/outages',
+              statusCode: 401
+            }, done);
+          });
+        });
+      });
+    });
+
+    describe('with an authenticated normal user', function () {
+      before(function (done) {
+        agent.get('/login/test/2').expect(200, done);
+      });
+
+      it('should require auth', function (done) {
+        storage.flush_database(function () {
+          storage.addService(validService, function (err, id) {
+            assert.ifError(err);
+            superAgentAssertions.shouldReturnStatusCode(agent, {
+              url : API_ROOT + '/services/outages',
+              statusCode: 401
+            }, done);
+          });
+        });
+      });
+    });
+
+    describe('with an authenticated admin user', function () {
+      before(function (done) {
+        agent.get('/login/test/1').expect(200, done);
+      });
+
+      it('should load services', function (done) {
+        agent
+          .get(API_ROOT + '/services/outages')
+          .set('Accept', 'application/json')
+          .expect('Content-Type', /json/)
+          .expect(200)
+          .send()
+          .end(function (err, res) {
+            // TODO: Insert some outages and check against the result body
+            done(err);
+          });
+      });
+    });
+  });
```

Comment:
We need tests that prove that you only get the outages from services you can access (if you are not admin)